packages/sentry/src/sentry.ts

```diff
@@ -38,6 +40,10 @@ export const boot = () => {
           onerror: false,
         }),
       ],
+      beforeSend(event) {
+        // @ts-expect-error
```

Comment:
The types seem wrong. Looking at 

1. https://github.com/odynvolk/omit-deep-lodash/blob/master/src/index.js#L25

We can see that it's a dynamic arguments processing given we have more than two we silce.

The types do not respect that with `declare function omitDeep(input: object, props: string): object;`. Either we:

1. Path upstream (takes time)
2. Locally type
3. Know about the error and opt out

Reply:
I'm wondering if we should just implement this function on our own, the implementation is just a bunch a lines.
Then we also don't have the issue with the types.

Reply:
Did you have something like 3b9b6b1 in mind?